logrus.go

```diff
@@ -140,4 +140,11 @@ type FieldLogger interface {
 	Errorln(args ...interface{})
 	Fatalln(args ...interface{})
 	Panicln(args ...interface{})
+
+	IsDebugEnabled() bool
```

Comment:
We can not change the FieldLogger, such an interface change may raise build breaks to some users.

Reply:
hmmm... in which case we can raise build breaks to some users ?
If in there build process they pull the master branch of logrus before building ?

Reply:
Indeed, that the case I was talking about. We can not lightly introduces non backward compatible change to master.